### PR TITLE
Use ActivateWindow to update container

### PR DESCRIPTION
**Pull request by mediaminister**
It seems this works better than `Container.Update`: https://github.com/pietje666/plugin.video.vrt.nu/issues/616

### CODE REVIEW

File: resources/lib/kodiutils.py [hunk 1]
```diff
@@ -694,8 +694,11 @@ def container_refresh(url=None):
 def container_update(url=None):
     ''' Update the current container with respect for the path history. '''
     if url:
-        log(3, 'Execute: Container.Update({url})', url=url)
-        xbmc.executebuiltin('Container.Update({url})'.format(url=url))
+        # log(3, 'Execute: Container.Update({url})', url=url)
+        # xbmc.executebuiltin('Container.Update({url})'.format(url=url))
+        # ActivateWindow seems to work better after closing Kodi Player
+        log(3, 'Execute: ActivateWindow(Videos,{url})', url=url)
+        xbmc.executebuiltin('ActivateWindow(Videos,{url})'.format(url=url))
```
Comment (dagwieers) [inline resources/lib/kodiutils.py]: Should this not use `ReplaceWindow()` instead? Let me test...

Comment (dagwieers) [reply resources/lib/kodiutils.py]: Also, I fear that if the container has changed, the user may suddenly end up in a different container. So in this case the protection better works 100% !

Comment (dagwieers) [reply resources/lib/kodiutils.py]: The problem is with all our improvements I cannot make this do what we expect it to.
It never gets triggered, the Kodi container refresh always works correctly now.

Comment (dagwieers) [reply resources/lib/kodiutils.py]: Hmm, after testing this using `kodi-send` this is not doing anything if the URL provided is exactly the same as the virtual directory the we have open.

In other words, it does the opposite of what we want. We want to refresh the container only when it is the exact same URL as is currently active.

Comment (dagwieers) [reply resources/lib/kodiutils.py]: Also, there appears no difference in `ActivateWindow()` and `ReplaceWindow()`.
According to the documentation `ActivateWindow()` adds a new window to the history, and `ReplaceWindow()` does not. The effect of both is that if you go back, you go to the parent of the parent. So you are replacing the parent in the history, not the current entry. :disappointed:

Comment (mediaminister) [reply resources/lib/kodiutils.py]: I see, I'll close this.

**Pull request closed without merging**